### DISCUSSION TITLE
Addon needs to be refreshed for each blender session.

### DISCUSSION THREAD

**Discussion by loicbramoulle** [tingjoybits/Brush_Manager#4] [General]
Thanks for sharing this, definitely helps with making sculpting in blender more accessible.
The only issue I have is that the panel where I can pick the brushes from the folder I defined in the preferences, is never there when a new blender session is launched, unless I un-load, re-load the plugin, re-set the brush folder. then it appears again.
I'm on 2.92.
I'm purchassing the PowerManage addon from blender market now to see if it can allow to make BrushManager appear in blender in a couple clicks.
Thanks.

**Comment by tingjoybits** [comment]
Are you talking about custom categories that located in the library path, right? Does the path string is persistent in the preferences when a new blender session is launched? It shouldn't be an issue. Maybe you should try to load factory settings and use a default factory scene file to pinpoint the potential conflicts in files or settings.

**Reply by loicbramoulle** [reply]
Thanks for your reply. here is on startup, BrushManager not being displayed:
![Capture d’écran 2021-04-11 160040](https://user-images.githubusercontent.com/49880198/114317659-f4326b80-9ae7-11eb-83fe-da4886acbae0.png)
And by un-loading, then re-loading the addon in the pref, BrushManager appears again:
![Capture d’écran 2021-04-11 1600040](https://user-images.githubusercontent.com/49880198/114317697-1af0a200-9ae8-11eb-8297-f747bbcaa89c.png)
But then the library folder with the brushes I grabbed from the web needs to be re-selected in the addon preferences as it was un-loaded. (it's persistent between sessions, but BrushManager is invisible if not re-loaded/reset.)
Thanks.
Blender seems to be a real alternative to Zbrush now, but without somethign like brush manager it's soo obscure how to simply have something else than the default 10ish brushes. 
(append files to default scene with fake users and absolute path blabla, what happens when you want to add or remove brushes while you work and have these changes in your brush library be perpetual like in any sculpting app ect..?)

**Comment by tingjoybits** [comment]
Hmm, I never experienced such behavior of this add-on... Which OS are you using? Is there any errors in the system console window? Did you try with factory settings? Maybe there is a duplicate of the add-on in the scripts folder.

**Reply by loicbramoulle** [reply]
Ah Classic®, it seems to be working now, althought I tried dozens of time the last days. What I did today is clean-up the number of loaded addons I have, thanks to PowerManager, from 15ish (not so much) to 5ish, it's the only thing I did I believe.
PowerManager saves the library file path into a Json file, so I tried to make BrushManager re-appear in a couple click on the eye icon in PowerManager, and it worked all fine, BrushManager did re-appear in a new session. 
But now even a new session has BrushManager working file without re-loading it.
Thanks for the messages, hope it helps in case someone else has this issue.
(I'm on Win10, no errors on system console it seems.)

**Reply by tingjoybits** [reply]
Glad you managed to solve it, cheers, and take care.

**Reply by Veberes23** [reply]
Hey, I have the same issue)
